android/src/main/java/org/devio/rn/splashscreen/SplashScreen.java

```diff
@@ -112,6 +112,14 @@ public void onCompletion(MediaPlayer mp) {
                             hideVideo(activity);
                         }
                     });
+
+                    videoView.setOnErrorListener(new MediaPlayer.OnErrorListener() {
+                        @Override
+                        public boolean onError(MediaPlayer mp, int what, int extra) {
+                            Log.d("splashscreen video", "onError" + Integer.toString(what) + ", " + Integer.toString(extra));
+                            return true;
```

Comment:
should we return false to cause the onCompletion listener to be called and hide the video?

Reply:
2ab282c72554b9a865660c91b6dcf424a8f91db2